tox.ini

```diff
@@ -7,6 +7,7 @@ passenv = LANG LC_CTYPE LC_ALL MSYSTEM
 deps =
     mock
     nose
+    toml
 commands =
```

Comment:
Then you wouldn't need to add the dependency here.

Reply:
Do I need it in the other locations? I seem to be struggling with this...

Reply:
No, if you have it in the extra, it should be installed into all toxenvs.